packages/quantic/force-app/main/default/lwc/quanticResultCopyToClipboard/quanticResultCopyToClipboard.js

```diff
@@ -0,0 +1,126 @@
+import copied from '@salesforce/label/c.quantic_Copied';
+import copy from '@salesforce/label/c.quantic_Copy';
+import {
+  registerComponentForInit,
+  initializeWithHeadless,
+  getHeadlessBundle,
+} from 'c/quanticHeadlessLoader';
+import {copyToClipboard, buildTemplateTextFromResult} from 'c/quanticUtils';
+import {LightningElement, api} from 'lwc';
+
+/** @typedef {import("coveo").Result} Result */
+/** @typedef {import("coveo").InsightEngine} InsightEngine */
+
+/**
+ * The `QuanticResultCopyToClipboard` component allows the end user to copy a result to clipboard.
+ * @category Insight Panel
+ * @example
+ * <c-quantic-result-copy-to-clipboard engine-id={engineId} result={result} label="Copy" success-label="Copied!" text-template="${title}\n${clickUri}"></c-quantic-result-copy-to-clipboard>
```

Comment:
`<c-quantic-result-copy-to-clipboard engine-id={engineId} result={result} label="Copy" success-label="Copied!" text-template="${title}\n${clickUri}"></c-quantic-result-copy-to-clipboard>`

`<c-quantic-result-copy-to-clipboard engine-id={engineId} result={result} text-template="${title}\n${clickUri}"></c-quantic-result-copy-to-clipboard>`

You don't need the values for the labels if you have a default copy/copied no?

Reply:
yeah indeed, I added the values for the label just to showcase how these properties can be used. I will change the values to custom values to make more sense.

Reply:
Yea I just feel like if already have values for these labels and they're also translated, we shouldn't "encourage" the user to change them.